test/test-moveTetromino.js

```diff
@@ -738,7 +802,9 @@ describe("Move tetromino", function () {
         `)
     })
     it("O tetromino can not move, item too down", function () {
```

Comment:
your test says that tetromino cannot move, but expected is different from coordinatesTetromino

Reply:
Also, where is assertion?